### ISSUE TITLE
Is video conferencing necessary?

### ISSUE THREAD

**Issue by timolinn** [Sedronik/remote-doctor#2]
@cyberdroidmann  I did a little research today on this. Actually I started looking at WebRTC samples and mini tutorials to get a feel of what we can do with it. Then it dawned on me that since Jitsi uses a Java Backend for it's <b>*video bridge*</b>, as the name sounds, the software bridges multiple video streams in to one.

This raised the question: Do we need a videobridge on remote doctor?

**Comment by bitsofsteve** [comment]
> @cyberdroidmann I did a little research today on this. Actually I started looking at WebRTC samples and mini tutorials to get a feel of what we can do with it. Then it dawned on me that since Jitsi uses a Java Backend for it's _video bridge_, as the name sounds, the software bridges multiple video streams in to one.
> 
> This raised the question: Do we need a videobridge on remote doctor?

Considering the idea of just one to one communication and a minor need for signalling(sending messages) which webRTC does not specify but we can use services like Socket.io for it, WebRtc can sure stand in, as long as it has protocols for stream and data exchange.

Good resources I found:

https://codelabs.developers.google.com/codelabs/webrtc-web/#0
https://developer.mozilla.org/en-US/docs/Web/API/WebRTC_API

**Comment by timolinn** [comment]
Yes! Funny I didn't imagine the web sockets part, given that WebRTC literally means  (Web Real-Time Communication) 😄, Thanks for the resource super helpful.

**Comment by timolinn** [comment]
I once studied XMPP protocols, wondering if it'll be a better option than using web sockets?

**Comment by bitsofsteve** [comment]
> I once studied XMPP protocols, wondering if it'll be a better option than using web sockets?

Well, I'm not sure, let's keep researching. we have 3 days from research to understanding implementation
